include/rdma/fi_domain.h

```diff
@@ -52,6 +52,7 @@ extern "C" {
 #define FI_SYMMETRIC		(1ULL << 59)
 #define FI_SYNC_ERR		(1ULL << 58)
 #define FI_UNIVERSE		(1ULL << 57)
+#define FI_AUTH_KEY_MATCH_ALL	(1ULL << 56)
```

Comment:
That's fair. Let me run this pass the DAOS/Mercury folks since this was a potential way to easy their integration with this API.  But, removing this I think opens up another optimization for FI_SOURCE_ERR and the fi_cq_err_entry (I'll start new comment for this).

Reply:
Synced with DAOS/Mercury team. Seems to be OK to remove magic insert flag.